### ISSUE TITLE
Custom input size

### ISSUE THREAD

**Issue by MoaazAbdulrahman** [XuJiacong/PIDNet#13]
Thank you for your great effort. 

I am getting the following error when I run inference on image with different size than Cityscapes dataset. 
`RuntimeError: The size of tensor a (110) must match the size of tensor b (109) at non-singleton dimension 2`'

Should I resize the image before passing it to the model?

**Comment by XuJiacong** [comment]
Yes, and I think the input resolution should be at least divisible by 8.
